src/assignment/views/AssignmentEditMarcom.tsx

```diff
@@ -100,11 +100,16 @@ const AssignmentEditMarcom: FC<AssignmentEditMarcomProps & RouteComponentProps &
 				tText('assignment/views/assignment-edit-marcom___publiceren-naar-klascement-gelukt')
 			);
 		} catch (err) {
-			ToastService.danger(
-				tText(
-					'assignment/views/assignment-edit-marcom___publiceren-naar-klascement-mislukt'
-				)
+			const avoError = tText(
+				'assignment/views/assignment-edit-marcom___publiceren-naar-klascement-mislukt'
 			);
+			const klascementError =
+				(err as any)?.innerException?.additionalInfo?.responseBody?.additionalInfo
+					?.klascementError?.uitzonderingen?.[0]?.diagnose ||
+				(err as any)?.innerException?.additionalInfo?.responseBody?.additionalInfo
+					?.klascementError;
```

Comment:
since we use it in 2 locations it might be good to have a helper:
getReadableKlascementErrorMessage(err): string